### DISCUSSION TITLE
重启nginx失败

### DISCUSSION THREAD

**Discussion by jocinuly** [XTLS/Xray-core#3017] [Q&A]
![{DBE731C2-8AED-46a3-897F-38BB533F022E}](https://github.com/XTLS/Xray-core/assets/89686932/9619b895-ffeb-4a42-826d-55bcde5e5246)
![image](https://github.com/XTLS/Xray-core/assets/89686932/ccf44370-f857-4fef-b2f2-d5723e086f31)

**Comment by qwerr0** [comment]
老实说, 这个是你的Nginx配置文件写错了(第98行括号未闭合或者是少写了分号), 和Xray关系不大, 要解决应该把你的 nginx.conf 发出来
